openedx/core/djangoapps/theming/template_loaders.py

```diff
@@ -27,21 +27,21 @@ class ThemeFilesystemLoader(FilesystemLoader):
     is_usable = True
     _accepts_engine_in_init = True
 
-    def get_template_sources(self, template_name, template_dirs=None):
+    def __init__(self, engine, dirs=None):
```

Comment:
Ah, I'd missed that it's still possible to initialize the dirs with the loader instead of having to use the ones from the engine.  I think this should work correctly; it's also possible to pass the list to the loader from the settings file (see "loaders" under [here](https://docs.djangoproject.com/en/2.2/topics/templates/#django.template.backends.django.DjangoTemplates)), but this approach minimizes the changes from what we already had.

Reply:
thanks for the reference.